src/style.css

```diff
@@ -1 +1,58 @@
 /* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url('./fonts/Avenir.ttc');
+  font-weight: 300;
+}
+
+body {
+  padding: 0;
+}
+
+.search {
+  margin-top: 20px;
+}
+
+.search__input {
+  font-family: Avenir, sans-serif;
+  width: 100%;
+  box-sizing: border-box;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  background-image: url('/src/images/Search.svg');
```

Comment:
Verify the path for images in your CSS. It should match the required path `src/images`. Ensure that the image path is correct and the image exists at the specified location.